README.rst

```diff
@@ -166,6 +166,24 @@ Other available widgets:
 - ``itkwidgets.checkerboard``: Compare two images in a checkerboard pattern.
 - ``itkwidgets.compare``: Compare two images side-by-side.
 
+Using within a Docker Container
+-------------------------------
+You can use ``itkwidgets`` from within a docker container with jupyterlab.
+To create a local docker image:
+
+Install ``docker`` and build the docker image with::
```

Comment:
It would be really awesome to publish this docker image on Dockerhub. @thewtex, is there an ITK or ISC docker organization you could publish this under?

This would enable us to use this image as a base for another PyVista image we could publish

Reply:
@banesullivan  We have https://hub.docker.com/orgs/insighttoolkit, what is your docker id or email address ?

Reply:
My ID on DockerHub is the same as GitHub: `banesullivan`

Reply:
All set, you have been added

Reply:
We can also set up CI to push on merge to `master`.